goss.yaml

```diff
@@ -20,3 +13,24 @@ file:
     mode: "0755"
     owner: root
     group: root
```

Comment:
Should these be updated according to:
```yaml
        owner: 584792
        group: 584792
```
?

And if the test passed with these assertions in place then maybe we have a problem with our `organize`?